pandas/util/_decorators.py

```diff
@@ -65,8 +65,9 @@ def deprecate_kwarg(old_arg_name, new_arg_name, mapping=None, stacklevel=2):
     ----------
     old_arg_name : str
         Name of argument in function to deprecate
-    new_arg_name : str
-        Name of preferred argument in function
+    new_arg_name : str | None
+        Name of preferred argument in function. Use None to raise warning that
+        ``old_arg_name`` keyword is deprecated.
     mapping : dict or callable
         If mapping is present, use it to translate old arguments to
```

Comment:
could you add a couple of examples to the doc-string here? e.g. doing a rename and removing a kwarg

Reply:
I added a couple examples for removing the keyword.  

I did see a couple good examples of how to raise a warning when a keyword will be deprecated for another:
[keyword to keyword](https://github.com/pandas-dev/pandas/pull/19751/files#diff-e41db70762f6c5b076f9b551651d4132R80)
[keyword map (multiple keyword deprecations)](https://github.com/pandas-dev/pandas/pull/19751/files#diff-e41db70762f6c5b076f9b551651d4132R92)

Do let me know if you think we should still add some more examples for renaming kwargs.